### ISSUE TITLE
Notifications doesn't work

### ISSUE THREAD

**Issue by FiftyFour7250** [PiotrMachowski/Home-Assistant-Taskbar-Menu#20]
Hey, I've been looking for something that can send persistent notification to my desktop for a long time. And finally, I found your app. But it doesn't send :c
Your app isn't even in Windows "Notification and Action" settings
Can you please help me fix it?
I used the exe installer.
Now I'm on Windows 10 Education 21H2, but yesterday I had the Pro version, and notifications didn't work there either.

**Comment by PiotrMachowski** [comment]
What is displayed when you open a menu (with right mouse button)?
Have you turned on notifications mirroring?
![image](https://user-images.githubusercontent.com/6118709/127457483-94233dc2-4833-4f21-a062-d3bad76c6d00.png)

**Comment by FiftyFour7250** [comment]
> What is displayed when you open a menu (with right mouse button)?
> Have you turned on notifications mirroring?
> ![image](https://user-images.githubusercontent.com/6118709/127457483-94233dc2-4833-4f21-a062-d3bad76c6d00.png)

Yea, it was enabled by default, I also tried to turn it off, restart, and turn it back on, but it still doesn't work(

![image](https://user-images.githubusercontent.com/78786088/127523411-55b660d4-1119-4cd1-b4ff-94a453ae67c1.png)

**Comment by FiftyFour7250** [comment]
Help me please. I really really need this. Maybe I can provide some additional information? I use [portable version by AlexxIT](https://github.com/AlexxIT/HassWP), but I don't think that persistent notification is different in it

**Comment by PiotrMachowski** [comment]
@FiftyFour7250 I have no idea why it doesn't work for you. How do you create persistent notification?

You can use exe with "DEBUG" in name to check what happens in taskbar menu internally, maybe it will tell something

**Comment by FiftyFour7250** [comment]
@PiotrMachowski I don't know. Receiving notifications always works perfectly, but for some reason windows notifications don't arrived, and only from your app. Other apps, including apps from github, show windows notifications. I understand that you will not redo something just for me, but I think html5 notifications would be better anyway

**Comment by mat4444** [comment]
Have you verified if the notifications from HA taskbar are allowed in the Windows Notification Center ? Perhaps it's disabled by default or you disabled it by mistake.
In my case, it works fine, they stay in the notification center - see screenshot in the attached link
[Screenshot](https://i.ibb.co/TYp2gGr/Sans-titre.png)

**Comment by FiftyFour7250** [comment]
@mat4444 

As I said, I don't even have it there.

![image](https://user-images.githubusercontent.com/78786088/127787967-9dbac578-5450-4337-8521-17896d714fa1.png)

But not everything is displayed there. There are applications that also not in this list, but windows notifications arrive perfectly.

**Comment by FiftyFour7250** [comment]
> @mat4444
> 
> As I said, I don't even have it there.
> 
> ![image](https://user-images.githubusercontent.com/78786088/127787967-9dbac578-5450-4337-8521-17896d714fa1.png)
> 
> But not everything is displayed there. There are applications that also not in this list, but windows notifications arrive perfectly from them.

**Comment by PiotrMachowski** [comment]
@FiftyFour7250 it should appear there after a first successful notification. Try using this service call:
```
service: persistent_notification.create
data:
  title: Test title
  message: Test message
```

**Comment by FiftyFour7250** [comment]
@PiotrMachowski 
Nothing :c

**Comment by hode22** [comment]
yeah , i have the same issue here . maybe a windows update screwed notifications up .

**Comment by PiotrMachowski** [comment]
I'm not really sure where is the problem, everything works in my setup...

**Comment by hode22** [comment]
oooh it works :X:X:X this is so cool . i just reinstalled it and that seems to have done it :D

**Comment by PiotrMachowski** [comment]
<img src="https://media.giphy.com/media/26ufdipQqU2lhNA4g/source.gif"></img>

**Comment by FiftyFour7250** [comment]
Too bad it didn't work for me((

I have already reinstalled it about 10 times(

**Comment by mkono87** [comment]
So another abandoned project?

**Comment by PiotrMachowski** [comment]
@mkono87 ???

**Comment by mkono87** [comment]
@PiotrMachowski Sorry man I just saw the last reply was beginning of October and no update. Seen a lot of projects just go stale.

**Comment by PiotrMachowski** [comment]
@mkono87 I was not able to reproduce this problem, so currently there's nothing I can do.
I have a lot of repositories to support and my time is limited, so I'm not able to immediately solve all issues/add all features.

**Comment by mat4444** [comment]
The notifications worked for me for a while, but recently, I noticed they don't get replicated on my Windows 10 PC (I don't remember exactly the first occurrence date). Is there anything I can do to troubleshoot that ? Is it possible that a Windows 10 update messed up with the settings ?
Thanks :)

**Comment by PiotrMachowski** [comment]
@mat4444 I don't know what can cause these issues, they worked perfectly on my PC, then stopped working for some time and came back again later...

**Comment by kszere** [comment]
I have also this problem. Windows 11 Pro.
